Utilities/priority_queue.py

```diff
@@ -27,7 +29,9 @@ def push(self, priority, item):
             The data being inserted into the queue.
 
         """
-        heapq.heappush(self._queue, (priority.value, item))
+        val = int('{}{}'.format(priority.value, PriorityQueue.count))
+        PriorityQueue.count = PriorityQueue.count + 1
```

Comment:
Use compound operators where applicable
```suggestion
        PriorityQueue.count += 1
```